functions/controllers/applicants/index.js

```diff
@@ -0,0 +1,42 @@
+const functions = require("firebase-functions");
+const express = require("express");
+const cors = require("cors");
+const { jwtCheck, hasPermission } = require("../../utils/middleware");
+const { queryDocument } = require("../../utils/database");
+
+const applicant = express();
+applicant.disable("x-powered-by");
+applicant.use(express.json());
+
+const auth0Config = functions.config().auth;
+const corsConfig = auth0Config ? auth0Config.cors : "";
+
+const corsOptions = {
+  origin: corsConfig,
+  optionsSuccessStatus: 200,
+};
+
+applicant.use(cors(corsOptions));
+
+applicant.get("/applicant/:id/", jwtCheck, hasPermission, async (req, res) => {
+  try {
+    const snapshot = await queryDocument("applicants");
```

Comment:
queryDocument specifies an ID which should be from your request param I believe

Reply:
Ahh I had that and then removed it during debugging. I went ahead and pushed up the changes to both that and updated the permissions to `hasReadAdmin` (wasn't sure which one to use). Thanks!